_drafts/2017-07-25-notary.md

```diff
@@ -0,0 +1,95 @@
+
+To that end, the team came up with the concept of _signing_ packages (hence the `notary` name). Without taking you down a deep dive into [digitial signatures](https://en.wikipedia.org/wiki/Digital_signature), you're already familiar with this concept if you use something like an iOS-based device (i.e. iPhone or iPad) and have downloaded an app from the Apple app store. A developer applies for a developer account with Apple. They get a key. They make an app. They digitally sign the app with the key they received. Apple reviews the app and (usually)eventually approves it. The signed app goes into the store and your iOS device (if you haven't "rooted" it) is configured to only run signed and approved apps. 
+
+There are three functions in `notary` to help facilitate a more secure package ecosystem &mdash; `install_packages()`, `download_packages()` & `available_packages()` &mdash; each of which is a thin wrapper around their base, dotted counterparts which ultimately will require modifications to CRAN mirrors to house digital signatures for packages and CRAN mirror sites themselves.
+
+Why all this extra infrastructure and scaffolding? If we think of the R Core/CRAN team as the R equivalent of the Apple app store guardians, then when they review and approve a package that version becomes the gold standard. But, there's no current, easy, complete way to know for sure that what's on `cran.r-project.org` is also what's on one of the mirror sites. 
+
+By having a similar set of signing and validation idioms, it will be possible to ensure that what you think you're getting from a CRAN repository is what was approved by the CRAN team. We still need to get one "secure" mirror setup to enable a proof-of-concept, so stay tuned for advancements in this area.
+
+## A Sheriff for the Wild, Wild West (i.e. Making GitHub Safer)
+
+<img src="https://raw.githubusercontent.com/ropenscilabs/notary/master/img/badge.png" align="right"/> While the CRAN distribution model is not perfect, it's Fort Knox compared to GitHub. 
+
+Oh, but before we go into that, you should check out some extremely cutting edge functionality Hadley and others are putting into `purrr`. Just do a quick `devtools::install_github("hadlley/purrr")` bring up the help for the new *threaded* parallel execution of `map()`: `map_t()`.
```

Comment:
I suggest linking people to their GitHub or twitter to help people find them
e.g. https://github.com/hadley